pnpm-workspace.yaml

```
@@ -0,0 +1,3 @@
+packages:
+  - "apps/*"
+  - "packages/*" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)